Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardEditViewModel.swift

```diff
@@ -7,35 +7,109 @@ import SwiftUI
 import Common
 import Storage
 
+enum CreditCardEditState: String, Equatable, CaseIterable {
+    case add
+    case edit
+    case view
+
+    enum leftBarButtonState: String, Equatable, CaseIterable {
+        case Close
```

Comment:
I'm not sure if there is a Swiftlint rule but we create the cases for the enums with lowercase so `close` instead of `Close`

Reply:
Good catch, added this while testing something and this indeed should be lower cased